### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S "./.git,*.pdf,*.po,*.pot,*.ts" -L abl,addess,afe,als,atleast,ba,corrently,datas,ist,plattform,pres,repid,sie`

### CODE REVIEW

File: io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties [hunk 1]
```diff
@@ -405,7 +405,7 @@ App.TimeOfUseTariff.Tibber.accessToken.label = Token
 App.TimeOfUseTariff.Tibber.accessToken.description = To link to your Tibber account you need a personal access token. You can create this under "developer.tibber.com/settings/access-token".
 App.TimeOfUseTariff.Tibber.filterForHome.label = Filter for Home
 App.TimeOfUseTariff.Tibber.filterForHome.description = For multiple 'Homes', add either an ID (format UUID) or 'appNickname' for unambiguous identification
-App.TimeOfUseTariff.Tibber.multipleHomesCheck.label = Do you have more than one contract connected to�your Tibber account?
+App.TimeOfUseTariff.Tibber.multipleHomesCheck.label = Do you have more than one contract connected to�your Tibber account?
```
Comment (luzpaz) [inline io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties:408]: Looks like vscodium incorrectly introduced this change

Comment (luzpaz) [reply io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties:408]: Message to merge dev: I just blew up my branch trying to correct this but haven't committed anything yet. Is it possible for you to cherry pick all the rest of the edits but drop the changes to `io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties` please ?

Comment (luzpaz) [reply io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties]: :pray:

Comment (sfeilmeier) [reply io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties]: No problem. Thanks for the contribution again!

You can always reset a file to `develop` via `git checkout develop io.openems.edge.core/src/io/openems/edge/core/appmanager/translation_en.properties`

**Pull request merged**